content/espanol/python-turtle/activity-10.md

```diff
@@ -5,9 +5,9 @@ weight: 13
 draft: false
 ---
 
-![mandala flower with random color petals](https://paper-attachments.dropbox.com/s_F078714AB8FA59FD292476DB5E3304D54CD817148B6A8087A90D63D6C59A7C0D_1563989900782_mandala+colors.PNG)
+![alt text width="70%"](../media/mandala-color.png "mandala flower with random color pedals")
```

Comment:
Can you look back and translate all the alt texts too? (e.g. here, translate "mandala flower with random color pedals")

Reply:
updated them all (afaik)